clang/lib/CodeGen/CodeGenModule.h

```diff
@@ -26,6 +26,7 @@
 #include "clang/Basic/LangOptions.h"
 #include "clang/Basic/Module.h"
 #include "clang/Basic/NoSanitizeList.h"
+#include "clang/Basic/PointerAuthOptions.h"
```

Comment:
This include looks unneeded since further forward declaration `class PointerAuthSchema;` does the job